src/state/urls.js

```diff
@@ -0,0 +1,6 @@
+export function getMetricURL(appName, metricName) {
+  // workaround servers that incorrectly interpret url resources with a "." in
+  // them as having an extension (this is common for metrics) -- the glean
+  // schema doesn't allow `-`'s in metrics, so this should be ok
+  return `/apps/${appName}/metrics/${metricName.replace(/-/g, "i")}`;
```

Comment:
This is odd to me... you're replacing hyphens with the string `i`?

Reply:
I messed this up while adding a test. 🤦 Should be good now.